### PR TITLE
Ft inventory

### PR DESCRIPTION
**Pull request by mjedwards**
## Description
Describe change or new feature here.
## Type of change
Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Checklist
Remove any items which are not applicable.

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project

### CODE REVIEW

File: src/components/Navbar.js
Comment (erferguson) [inline src/components/Navbar.js]: can probably be removed?

File: src/components/Navbar.js
Comment (erferguson) [inline src/components/Navbar.js]: can probably be removed?

File: src/components/Navbar.js
Comment (erferguson) [inline src/components/Navbar.js]: can be removed?

File: src/components/Navbar.js
Comment (erferguson) [inline src/components/Navbar.js]: can be removed?

File: src/components/Navbar.js
Comment (erferguson) [inline src/components/Navbar.js]: 97-101, can be removed?

File: src/components/Orders/Orders.js
Comment (erferguson) [inline src/components/Orders/Orders.js]: doesn't appear that useSelector is being used in the code base

File: src/components/Orders/Orders.js
Comment (erferguson) [inline src/components/Orders/Orders.js]: Tag and Button not called in code

File: src/components/Orders/Orders.js
Comment (erferguson) [inline src/components/Orders/Orders.js]: connect not called in code, remove?

File: src/components/SellerAccount/SellerAccount.js
Comment (erferguson) [inline src/components/SellerAccount/SellerAccount.js]: remove if not using

**Review by erferguson — CHANGES_REQUESTED** [review]
just minor edits re: removing commented out code. good work, Mike!

**Comment by mjedwards** [comment]
made the suggested corrections

**Review by Cireimu — CHANGES_REQUESTED** [review]
Personally cannot approve this right now. It seems to be just a quick thing of flexbox or something but items just float off the screen, also have a ton of horizontal and the sizes vary drastically. Also, images should have standard sizing with max-width, upon loading this page I had two items that took up the entire screen. I can work with you on these things if needed but I need this to be fixed first before I can merge them.  
inventory screen

**Comment by mjedwards** [comment]
Everything should be fixed now.

**Pull request merged**
